### DISCUSSION TITLE
Manaswini Bingi

### DISCUSSION THREAD

**Discussion by Manaswini-15** [github/india#333] [Students]
I didn't attend the orientation session. How can I get the recorded session link? 
I am unable to understand about the internship completely could u help me out of this problem.

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together!
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏

**Comment by ajeetkbhardwaj** [comment]
> I didn't attend the orientation session. How can I get the recorded session link? I am unable to understand about the internship completely could u help me out of this problem.

Just login to your FRT account and go to orientation session and scrolled down at the end you saw the recorded video presented there
